clients/roscpp/include/ros/node_handle.h

```diff
@@ -112,11 +112,17 @@ namespace ros
     /**
      * \brief Parent constructor
      *
-     * This version of the constructor takes a "parent" NodeHandle, and is equivalent to:
+     * This version of the constructor takes a "parent" NodeHandle.
+     * If the passes ns is relative (does not start with a slash), it is equivalent to calling:
```

Comment:
Fixed the four locations of `passes`.

I would rather keep `ns` since it refers to the passed argument `ns`. `Namespace` could either mean `ns` or the namespace of the parent node handle and I think is therefore ambiguous.